api/internal/states/install/states.go

```diff
@@ -0,0 +1,42 @@
+package states
```

Comment:
This is what I came up as a way to share the state definitions between linux/kubernetes and app.